docs/guides/mobile-plugin-gma-ios.md

```diff
@@ -0,0 +1,78 @@
+---
+title: EUID GMA Plugin for iOS
+description: A guide for publishers using GMA for ad requests on iOS apps.
+hide_table_of_contents: false
+sidebar_position: 13
+---
+
+import Link from '@docusaurus/Link';
+
+# EUID GMA Plugin for iOS Integration Guide
+
+The EUID Google Mobile Ads (GMA) Plugin for iOS enables publishers that use the [Google Mobile Ads (GMA) SDK](https://developers.google.com/ad-manager/mobile-ads-sdk) to send <Link href="../ref-info/glossary-uid#gl-euid-token">EUID tokens</Link> as [secure signals](https://support.google.com/admob/answer/11556288) in ad requests from iOS apps. This is done automatically, with no direct coding needed from the app developer.
+
+## Functionality
+
+This plugin simplifies integration with Google Mobile Ads (GMA) for any publishers who want to support EUID for apps running on iOS devices.
+
+## Version
+
+<!-- As of 2024-04-17 -->
+
+This documentation is for the EUID GMA Plugin for iOS version 0.3.2 or later.
```

Comment:
IOS GMA version 1.0.0

Reply:
@caroline-ttd Done but should I also remove the "or later" here? I left it in for now.